manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: clusterlogging.v4.1.2
+  name: clusterlogging.v4.1.0
```

Comment:
Is it going to be a problem if there is already a CSV with `name: clusterlogging.v4.1.2` out there?  That is, will OLM ignore the CSV with `name: clusterlogging.v4.1.0` even though it is supposed to be "newer"?

Reply:
@richm this value is going to be substituted by the ART pipeline, it will not be published as "name: clusterlogging.v4.1.0", it will be published as "name: clusterlogging.v4.1.4" (and then 4.1.5, and then 4.1.6, etc).